### PR TITLE
Rework the constants for readability

### PR DESCRIPTION
**Pull request by JustinDrake**

### CODE REVIEW

File: specs/casper_sharding_v2.1.md [hunk 1]
```diff
@@ -34,27 +34,43 @@ Note: the python code at https://github.com/ethereum/beacon_chain and [an ethres
...
+| `MIN_COMMITTEE_SIZE` | 2**7 (= 128) | validators |
+| `MAX_VALIDATOR_COUNT` | 2**22 ( = 4,194,304) | validators |
+| `GENESIS_TIME` | **TBD** | seconds |
+| `SLOT_DURATION` | 2**4 (= 16) | seconds |
+| `CYCLE_LENGTH` | 2**6 (= 64) | slots | ~17 minutes |
+| `MIN_DYNASTY_LENGTH` | 2**8 (= 256) | slots | ~1.1 hours |
+| `SQRT_E_DROP_TIME` | 2**16 (= 65,536) | slots | ~12 days |
+| `WITHDRAWAL_PERIOD` | 2**19 (= 524,288) | slots | ~97 days |
+| `BASE_REWARD_QUOTIENT` | 2**15 (= 32,76) | — |
+| `MAX_VALIDATOR_CHURN_QUOTIENT` | 2**5 (= 32) | — |
```
Comment (ChihChengLiang) [inline specs/casper_sharding_v2.1.md]: Had difficulty in figuring out what "churn" means before.
Is the usage here in the same sense of losing a customer?
https://ethresear.ch/t/registrations-shard-count-and-shuffling/2129

Comment (JustinDrake) [reply specs/casper_sharding_v2.1.md]: Churn is just a slightly more precise term to the generic word "change". It means "change in a set of things".

Comment (ChihChengLiang) [reply specs/casper_sharding_v2.1.md]: Thanks for clarifying. ❤️

File: specs/casper_sharding_v2.1.md [hunk 2]
```diff
@@ -34,27 +34,43 @@ Note: the python code at https://github.com/ethereum/beacon_chain and [an ethres
...
+**Status codes**
+
+| Status code | Value |
+| - | :-: |
+| `PENDING_LOG_IN` | `0` |
+| `LOGGED_IN` | `1` |
+| `PENDING_EXIT` | `2` |
+| `PENDING_WITHDRAW` | `3` |
+| `WITHDRAWN` | `4` |
+| `PENALIZED` | `128` |
```
Comment (vbuterin) [inline specs/casper_sharding_v2.1.md]: Add `ENTRY = 1` and `EXIT = 2` (introduced in https://github.com/ethereum/eth2.0-specs/pull/24/files).

**Comment by djrtwo** [comment]
Handled merge conflict. Changes look good. merging!

**Pull request merged**